packages/devtools_app/lib/src/inspector/flutter/inspector_tree_flutter.dart

```diff
@@ -177,21 +203,30 @@ class _InspectorTreeState extends State<InspectorTree>
   Future<void> currentAnimateY;
   Rect currentAnimateTarget;
 
+  // fade-in or fade-out animation for Constraints when toggling Show Constraints button
+  AnimationController _animationController;
+
   @override
   void initState() {
     super.initState();
     _scrollControllerX = ScrollController();
     _scrollControllerY = ScrollController();
     _scrollControllerY.addListener(_onScrollYChange);
     _bindToController();
+    _animationController = AnimationController(
+      vsync: this,
+      duration: const Duration(milliseconds: 500),
+    );
   }
 
   @override
   void dispose() {
     super.dispose();
-    controller?.client = null;
+    controller?.removeClient(this);
+    controller?.animationController = null;
```

Comment:
remove the animation controller from the client. I think you aren't actually setting it on the client anyway unless I'm missing somethign.